### DISCUSSION TITLE
Checkbox selection not showing in features in QField

### DISCUSSION THREAD

**Discussion by OBVYamaska** [opengisch/QField#2888] [Q&A]
Small bug: if I have checkboxes in a form (with multiple selections option checked), I can select them without any issue.
But if I go back in an existing feature which has such selected boxes, they do not show.
If I sync the project, I can see the selections in QGIS, so they are effectively stored in the QField project.

This is still annoying, as my field workers sometimes need to verify previous surveys and all...

**Comment by larslund** [comment]
My coworkers and I have the same problem.
The form I provide contains multiple Value Relation Fields with "Allow multiple selections" (checkboxes) activated. In QField, checked boxes (which of course have been saved), disappear in any feature at the next display.

At the same I noticed this behaviour time I got log entries saying "Picture expression eval error" on project load which I firstly thought to be related. Searched for hours for the cause, dissected the project. I now believe this is a bug of QField, as in QGIS, as has been mentioned, everything looks good.

During field surveys, this leads to much insecurity, which data has been registered and whether the input is even saved at all.
